spec/unit/actions/synchronize_clients_spec.rb

```diff
@@ -68,6 +83,50 @@
       expect { subject.call(params) }.to raise_error(expected_error_message)
     end
   end
+
+  context 'when synchronizing client get errors' do
+    before do
+      detail_items = []
+      detail_items << {
+        status: 'error',
+        id: 'pid',
+        error: {
+          message: 'Error sync ds',
+          parameters: 'test_ds'
```

Comment:
used meaningful name instead of `test_ds`

Reply:
Done update code